client/my-sites/checkout/checkout-thank-you/business-plan-details.jsx

```diff
@@ -46,6 +46,17 @@ const BusinessPlanDetails = ( { selectedSite, sitePlans, selectedFeature } ) =>
 					href={ '/themes/' + selectedSite.slug } />
 			}
 
+			{ ! selectedFeature &&
+				<PurchaseDetail
+					icon="plugins"
+					title={ i18n.translate( 'Upload a Plugin' ) }
+					description={ i18n.translate(
+						'You can now upload your own plugins, or ones you\'ve downloaded, directly through a drag and drop interface.'
+					) }
+					buttonText={ i18n.translate( 'Upload a plugin now' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **68** times:
`translate( 'Upload Plugin' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).